### PR TITLE
[IMP] hr-holidays add dashboard

### PR DESCRIPTION
**Pull request by jbm-odoo**
Description of the issue/feature this PR addresses:
We need to use an international word to define holidays and leaves.
Employees needs a good dashboard and leaves summary

Desired behavior after PR is merged:
We have Time Off (no more leaves).
Each user has a intuitive dashboard for his Time Off.
The dashboard has a calendar and a summary of all time off and time off request.
Some intro messages are more clear and give correct information.

id=1916871
--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/hr_holidays/__manifest__.py [hunk 1]
```diff
@@ -47,6 +47,7 @@
         'report/hr_leave_reports.xml',
 
         'views/hr_views.xml',
+        'views/hr_leave_template.xml',
```
Comment (legalsylvain) [inline addons/hr_holidays/__manifest__.py]: hr_time_off_template.xml ?

Comment (tivisse) [reply addons/hr_holidays/__manifest__.py]: @legalsylvain Thanks for the feedback. 

According to guidelines, we use the model technical name, not the user one.

File: addons/hr_holidays/static/src/js/time_off_calendar.js [hunk 2]
```diff
@@ -0,0 +1,194 @@
...
+                    if (content.allocation_type != 'no') {
+                        leave_state += content.virtual_remaining_leaves
+                        leave_state += " / "
+                        leave_state += content.max_leaves
+                    }
+                    else {
+                        leave_state += -content.virtual_remaining_leaves
+                    }
+                    leave_state += '</nobr></h1>'
+                    leave_state += '<h3>' + leaves[leave][0] + '</h3>'
```
Comment (tivisse) [inline addons/hr_holidays/static/src/js/time_off_calendar.js]: Be careful. If the leave type name is ```<script>alert('Alert');</script>``` you will execute JS code.

File: addons/hr_holidays/views/hr_leave_type_views.xml [hunk 3]
```diff
@@ -55,10 +53,14 @@
                             <field name="allocation_type" widget="radio" force_save="1"/>
                         </group>
                         <group name="validity" string="Validity">
-                            <field name="validity_start"/>
-                            <field name="validity_stop"/>
+                            <label for="validity_start" string="From"/>
```
Comment (RomainLibert) [inline addons/hr_holidays/views/hr_leave_type_views.xml]: Don't put those fields next to each other. What was previously done was correct

File: addons/project_timesheet_holidays/models/hr_holidays.py [hunk 4]
```diff
@@ -16,26 +16,38 @@ def _default_task_id(self):
         company = self.company_id if self.company_id else self.env.user.company_id
         return company.leave_timesheet_task_id.id
 
-    timesheet_generate = fields.Boolean('Generate Timesheet', default=True, help="If checked, when validating a leave, timesheet will be generated in the Vacation Project of the company.")
-    timesheet_project_id = fields.Many2one('project.project', string="Internal Project", default=_default_project_id, help="The project will contain the timesheet generated when a leave is validated.")
-    timesheet_task_id = fields.Many2one('project.task', string="Internal Task for timesheet", default=_default_task_id, domain="[('project_id', '=', timesheet_project_id)]")
+    timesheet_generate = fields.Boolean('Generate Timesheet', default=True, help="If checked, when validating a time off, timesheet will be generated in the Vacation Project of the company.")
```
Comment (RomainLibert) [inline addons/project_timesheet_holidays/models/hr_holidays.py]: Why those changes?

**Comment by tivisse** [comment]
@robodoo r+

**Pull request closed without merging**